content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,139 @@ description: Istio 1.5 release notes.
+- xDS push optimize #18354.
+- Override locality load balancer settings per host #18406.
+- eds: do not trigger continuous full pushes when a pod is in crash loop #18574.
+- Disable redirect calls to app itself via envoy #19308.
+- add an `iptables` failure detector, default off #19534.
+- Istio: add consecutive `5xx` and gateway errors for outlier detection #19771.
+- **Improved** `EnvoyFilter` matching performance optimization #19786.
+- handle http proxy while generating routes #19919.
+- Allow non-intercepting proxies `CAP_NET_BIND_SERVICE` via annotation #20378.
+- Initial support for custom Istio topology label kind/enhancement #19192.
+- `istio-iptables`: Use `iptables-restore` by default #18847.
+- Pilot-agent for in-process Ingress SDS  #18999.
+- turn on protocol sniffing for inbound #18666.
+
+## Security
+
+- **Added** Beta authentication API. The new API separates peer (i.e mutual TLS) and origin (JWT) authentication into `PeerAuthentication` and `RequestAuthenticaiton` respectively. Both new APIs are workload-oriented, as opposed to service-oriented in alpha `AuthenticationPolicy`.
+- **Added** Authorization Policy supports deny action and exclusion semantics.
+- **Added** Auto mutual TLS is in Beta. The feature is enabled by default.
+- **Added** SDS is in Stable. SDS is enabled by default for identity provisioning.
+- **Improved** Node agent is merged with Pilot agent. This removes the requirement of pod security policy and improves security posture.
+- **Improved** Support two providers for Pilot (`istiod`) certificate: Kubernetes CA and Citadel.
+- **Improved** Support two types of JWT for CSR authentication: `first-party-jwt` and `third-party-jwt`.
+- **Improved** Provision key and cert to Prometheus through Istio Agent.
+
+## Telemetry
+
+- **Added** TCP protocol support for v2 telemetry.
+- **Added** gRPC response status code support in metrics/logs.
+- **Added** support for Istio Canonical Service.
+- **Improved** stability of v2 telemetry pipeline.
+- **Added** alpha-level support for configurable telemetry in v2 telemetry.
+
+Triage done:
+- **Added** support for populating AWS platform metadata in Envoy node metadata.
+- **Improved** Updated the Stackdriver adapter for Mixer to support configurable flush intervals for tracing data.
+- **Added** support for a headless collector service to the Jaeger addon.
+- Fixed `kubernetesenv` adapter to provide proper support for pods that contain a dot in their name.
+- Updated the Fluentd adapter for Mixer to provide millisecond-resolution in exported timestamps.
+
+- fix(bootstrap): add time-bounded platform detection logic with fail-fast #19971.
+- mixer: remove explicit flush in OpenCensus tracing adapter #18074.
+- mixer: Allow configuration of trace flush interval #18109.
+- Added jaeger-collector-headless service #18278.
+- Fix mixer source namespace attribute when dot in pod name  #19022.
+- Let Fluentd adapter encode timestamp with millisecond #19093.
```

Comment:
These can all be remove. They are covered by the above items.

Reply:
```suggestion
```